tests/scanner/audit/data/instance_network_interface_test_rules_1.yaml

```diff
@@ -0,0 +1,13 @@
+rules:
+  - name: all networks covered in whitelist
+    project: '*'
+    network: '*'
+    is_external_network: True
+    whitelist:
+      xpn-master: 
```

Comment:
just want to doublecheck here that the ids here (I think they are project ids?) are scrubbed.